.gitignore

```diff
@@ -76,3 +76,7 @@ docs/public/*
 node_modules/
 vendor/
 rootfs/
+
+
+# Envoy config captured during development
+envoy_*.json
```

Comment:
There is a `Temp` folder in the repo. Not sure whether it's designed for scenarios like this. We can use put the debug files there, and add `Temp` in gitignore.

If it's not designed for this case, then we can use things like `tmp\`.

Reply:
Not a bad idea. I'll make a tweak to this script to use this in an upcoming PR.